sql/step_a111_settings.sql

```diff
@@ -0,0 +1,88 @@
+-- Drop the existing table
+DROP TABLE IF EXISTS OH_SETTINGS;
+
+-- Create settings table structure
+CREATE TABLE OH_SETTINGS (
+  SETT_ID INT(11) NOT NULL AUTO_INCREMENT COMMENT 'Default generated ID',
+  SETT_CODE VARCHAR(50) NOT NULL,
+  SETT_CATEGORY ENUM('general', 'telemetry', 'security', 'dicom', 'accounting', 'reports', 'pharmacy', 'gui', 'experimental', 'application') NOT NULL DEFAULT 'general',
```

Comment:
Sorry, the order matters in ENUM fields.
Changed 'dicom' -> 'imaging' (more broad concept)
Removed 'experimental' -> APISERVER could be left outside 
```suggestion
  SETT_CATEGORY ENUM('general', 'application', 'gui', 'accounting', 'pharmacy', 'imaging', 'reports', 'security', 'telemetry') NOT NULL DEFAULT 'general',
```